src/channel_layout.c

```diff
@@ -305,357 +307,226 @@ static struct SoundIoChannelLayout builtin_channel_layouts[] = {
             SoundIoChannelIdBackRight,
             SoundIoChannelIdBackCenter,
         }
-    },
+    };
+    return &layouts[0];
 };
+static struct SoundIoChannelLayout *builtin_channel_layouts = make_builtin_channel_layouts();
```

Comment:
Does this run `make_builtin_channel_layouts` at compile time? If so, does that work in C11 too? I think I'd rather have builtin_channel_layouts be statically initialized data without running make_builtin_channel_layouts at runtime. We can ditch the fancy [x] = {} syntax, instead just making carefully sure that the order is lined up with the enum order.